TodoListService/Controllers/TodoListController.cs

```diff
@@ -60,8 +60,8 @@ public class TodoListController : ApiController
         //
         private static string aadInstance = ConfigurationManager.AppSettings["ida:AADInstance"];
         private static string tenant = ConfigurationManager.AppSettings["ida:Tenant"];
-        private static string clientId = ConfigurationManager.AppSettings["ida:ClientId"];
-        private static string appKey = ConfigurationManager.AppSettings["ida:AppKey"];
+        private static string clientId = ConfigurationManager.AppSettings["ida:ClientID"];
+        private static string appKey = ConfigurationManager.AppSettings["ida:Password"];
```

Comment:
In a way this is more explicit. But on the other hand:
- this is not consistent with the other samples.
- this is not what ASP.NET generates.